web-components/src/components/form/BasketPutForm/BasketPutForm.tsx

```diff
@@ -84,6 +105,7 @@ const BasketPutForm: React.FC<React.PropsWithChildren<FormProps>> = ({
             submitForm={submitForm}
             label="Put in basket"
           />
+          <BasketPutFormOnChange onBatchDenomChange={onBatchDenomChange} />
```

Comment:
Had to use a component to handle batch denom changes because `onChange` props on Formik components  (`Field`, `'Form`) is never triggered.